modules/mixcr/1.2/mixcr.smk

```diff
@@ -0,0 +1,320 @@
+            receptor_type = lambda wildcards: run_dict[wildcards.chain],
+            aux = reference_files("downloads/igblast/optional_file/human_gl.aux"),
+            gdv = lambda wildcards: (reference_files("genomes/no_build/igblast/database/imgt_human_" + receptor_dict[wildcards.chain] + "_v.ndb")).replace(".ndb",""),
+            gdj = lambda wildcards: (reference_files("genomes/no_build/igblast/database/imgt_human_" + receptor_dict[wildcards.chain] + "_j.ndb")).replace(".ndb",""),
+            gdd = lambda wildcards: (reference_files("genomes/no_build/igblast/database/imgt_human_" + receptor_dict[wildcards.chain] + "_d.ndb")).replace(".ndb",""),
+            run_flags = CFG["options"]["igblast_run"]["run_flags"],
+            form = "7 std btop " + CFG["options"]["igblast_run"]["form"]
+        conda:
+            CFG["conda_envs"]["igblast"]
+        shell:
+            op.as_one_line("""
+            igblastn -query {input.fasta} -out {output.db} 
+            -ig_seqtype {params.receptor_type} -organism human 
+            -auxiliary_data {params.aux} 
+            -germline_db_V {params.gdv} -germline_db_J {params.gdj} -germline_db_D {params.gdd} 
+            {params.run_flags} -outfmt '{params.form}' -domain_system imgt
+            """)
+
+    rule _update_mixcr_results:
+        input:
+            db = rules._igblastn_run.output.db,
+            og_mixcr = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.txt",
+            seq_info = rules._mixcr_to_fasta.output.seq_info,
+            script = CFG["igblast_scripts"]["igblastn2mixcr"]
+        output:
+            txt = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.status.txt"
+        shell:
+            "{input.script} -d {input.db} -m {input.og_mixcr} -o {output.txt} -s {input.seq_info}"
+
+    rule _symlink_mixcr_update:
+        input:
+            txt = rules._update_mixcr_results.output.txt
+        output:
+            txt = CFG["dirs"]["outputs"] + "txt/{seq_type}/mixcr.{sample_id}.clonotypes.{chain}.status.txt"
+        run:
+            op.relative_symlink(input.txt, output.txt, in_module=True)
+
+# Symlinks the final output files into the module results directory (under '99-outputs/')
+rule _mixcr_output_txt:
+    input:
+        txt = rules._mixcr_run.output.txt,
+        report = rules._mixcr_run.output.report
+    output:
+        txt = CFG["dirs"]["outputs"] + "txt/{seq_type}/mixcr.{sample_id}.clonotypes.ALL.txt",
+        report = CFG["dirs"]["outputs"] + "txt/{seq_type}/mixcr.{sample_id}.report"
+    run:
+        op.relative_symlink(input.txt, output.txt, in_module=True)
+        op.relative_symlink(input.report, output.report, in_module=True)
+
+rule _mixcr_chains:
+    input:
+        results = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.txt"
+
+
+# Generates the target sentinels for each run, which generate the symlinks
+
+if CFG["igblastn"]:
```

Comment:
I updated the `_mixcr_all` rules so that when `CFG["igblastn"] == True`, the extra file created by `_symlink_mixcr_update` is added to the required inputs